src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -128,7 +129,13 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                encodedAuthorization = "Basic "+new String(Base64.encodeBase64(authorization.getBytes()));
+                final Charset charset;
+                try {
+                    charset = Charset.forName("UTF-8");
```

Comment:
The Guava `Charsets` yes.

Do you know what the lowest supported JRE version must be?